### ISSUE TITLE
R7 250E missing from hardware support on wiki

### ISSUE THREAD

**Issue by Geroyuni** [Xaymar/OBS-AMD-Advanced-Media-Framework#401]
I'm not sure if I'm correct but [250E](https://www.techpowerup.com/gpu-specs/radeon-r7-250e.c2559) seems to have an encoder. It's a non-Oland GCN 1.0 card (like the 250X, and unlike the 250).

**Comment by Geroyuni** [comment]
Also, what's up with the Oland cards being in the list? I don't see any of them in the [Video Coding Engine wikipedia page](https://en.wikipedia.org/wiki/Video_Coding_Engine), but the github wiki inconsistently lists some, even ones that are rebrands of ones that are not listed or were shown to not have an encoder in this github

* Not listed: HD 8570, R5 240, R5 340X, R7 240, R7 350X
* R5 430 is listed but was shown to not be supported in #344
* R7 340 and R7 430 are listed but are rebrands of R7 240 (according to techpowerup). 240 was shown to not be supported as per #374
* HD 8670 and R7 435 are listed but I can't find any references to it being capable of encoding

Edit: Actually small exception I just noticed, the wikipedia page does list 2 Oland ones, HD 8570 which isn't in the github wiki, and HD 8670 which is. More confusing

**Comment by Xaymar** [comment]
The wiki lists cards that had a successful encoding submission, as well as data directly sourced from AMDs website at the time of writing. This also means that _only cards that were available or known at the time of writing are in the wiki_. Rumors and leaks were ignored.

**Comment by Geroyuni** [comment]
So to you, any of this 'Oland' stuff, or any information on wikipedia and techpowerup is not really anything looked into for the repository's wiki?

Given the way things are added here, what does this mean for listed cards in the wiki that were shown not to be able to encode in this repo's issues? Such as the R5 430 example given. And is R7 340 and R7 430 being rebrands of R7 240 considered inaccurate/unverified or insufficient given what has gotten them in the wiki?

I'm just trying to make sense of the tons of conflicting info online, plus the cards I've personally seen encode or fail to in other places.

**Comment by Xaymar** [comment]
I simply explained why it is in the current state, not what the requirements are for future updates. At the time, there was very little outside interest to maintain the Wiki at all, so I took what Information I had and compiled it into a somewhat readable page. As I've not maintained the plugin for almost a year now, I can't tell you what the current rules are, just what I did to maintain a reasonable level of correctness.
